### ISSUE TITLE
[Suggestion] max-players flag

### ISSUE THREAD

**Issue by TeQTabanca** [aromaa/WorldGuardExtraFlagsPlugin#216]
**I want to add a player restriction to a region, but I couldn't find this feature other than the "RedProtect" plugin. Any chance to add this feature?**

![image](https://user-images.githubusercontent.com/89253304/173148230-73a3f731-97ed-49fa-9906-cb3def9fd9ff.png)

**Comment by aromaa** [comment]
I think this meets the bar to be introduced. I don't like the idea of people being kicked out of the region but instead of have it as more of soft lock thing as if you login inside the region you may bypass the limit or if someone with bypass permission enter then they aren't counted towards the limit.

**Comment by aromaa** [comment]
No, there is no support for anything below 1.18. All of the code to support legacy versions of Minecraft has been removed, and there is no way to produce "separate jar" c30994bf42bfd6ef44ad8b533f4d500f5b43f513...aa149ab1ef001a854d5f5427295abfda8c6eb24e

**Comment by TeQTabanca** [comment]
I don't understand Java at all, but there are sites that convert .class files to .java. If it is done in a different way, it means there is nothing to be done. ([Source Code of Version 4.1.5](https://github.com/aromaa/WorldGuardExtraFlagsPlugin/files/8919701/WorldGuardExtraFlags_source_from_Procyon.zip))

**Comment by aromaa** [comment]
The support for 1.12.2 has been completely removed and I don't intend to bringing that back. All of the new features will be added to the latest version and it would be time consuming to backport these changes to older versions. If you want, you can hire a developer to do this for you.

**Comment by TeQTabanca** [comment]
**RedProtect plugin remained as the only method. Anyway, thank you for your time.  👍 **
